### PR TITLE
Updated continuous_systems.py with uniform_bar_modes function using Pycharm.

### PR DESCRIPTION
**Pull request by AnushaAnisetti**
Testing if Makefile executes automatically in Pycharm (in windows) and github interlink with Pycharm (in windows)

### CODE REVIEW

**Comment by josephcslater** [comment]
a) please update your branch(es) from master before submitting a pull request. It helps you resolve conflicts locally. 

b) Please see the readme.rst file in the code directory (adjacent to sdof.py) for some notes. 

c) numpy arrays-> array or arrays, per the numpy standard (array like if it can be another form of list.)

d) travis-ci checks (above) can be used to validate. It may have failed. Run pytest from the command line.

e) The editor doesn't need to be reported in the future, but glad to hear pycharm is working. Spyder may be easier for you to use as it is more "matlab" like. I don't know about git integration. 

f) looks good.

**Comment by josephcslater** [comment]
[Travis-ci](https://travis-ci.org/vibrationtoolbox/vibration_toolbox) tests the installation works. 

You should also occasionally try (make docs). 
 (welcome to the team)

**Comment by josephcslater** [comment]
I had to undo the merge because it failed tests on travis-ci.

**Comment by josephcslater** [comment]
Run pytest (type `pytest` at the command line) and make sure it passes your example test.

**Comment by josephcslater** [comment]
[https://travis-ci.org/vibrationtoolbox/vibration_toolbox](https://travis-ci.org/vibrationtoolbox/vibration_toolbox)

**Comment by AnushaAnisetti** [comment]
I am working on the errors. Thank you for checking.


Sincerely,

Anusha Anisetti, Ph.D.


________________________________

**Comment by josephcslater** [comment]
I think it was very minor: it's precise spacing in the output (or numbers). You can actually use the result from pytest (expected...) and put that in the docstring to make it work.

File: vibration_toolbox/continuous_systems.py [hunk 1]
```diff
@@ -90,6 +91,7 @@ def euler_beam_modes(n=10, bctype=2, npoints=2001,
     # len=[0:(1/(npoints-1)):1]';  %Normalized length of the beam
     len = np.linspace(0, 1, npoints)
     x = len * L
+    print(x)
```
Comment (raphaeltimbo) [inline vibration_toolbox/continuous_systems.py:93]: This print statement can make tests fail since this is not a value expected at the docstring.

**Pull request merged**